main/cloudwatch.tf

```diff
@@ -0,0 +1,160 @@
+          }
+       },
+EOF
+}
+
+data "template_file" "sqs_widgets" {
+  template = <<EOF
+       {
+          "type":"metric",
+          "width":12,
+          "height":6,
+          "properties":{
+             "metrics": [
+                 [ "AWS/SQS", "NumberOfMessagesSent", "QueueName", "${local.instance_alias}-sqs-queue" ],
+                 [ "AWS/SQS", "NumberOfMessagesReceived", "QueueName", "${local.instance_alias}-sqs-queue" ]
+             ],
+             "period":300,
+             "stat":"Average",
+             "region": "${var.aws_region}",
+             "title": "Shunting Yard SQS Sent & Received Messages"
+           }
+       },
+       {
+          "type":"metric",
+          "width":12,
+          "height":6,
+          "properties":{
+        	 "metrics": [
+               [ "AWS/SQS", "ApproximateNumberOfMessagesVisible", "QueueName", "${local.instance_alias}-sqs-queue" ],
+               [ "AWS/SQS", "ApproximateNumberOfMessagesDelayed", "QueueName", "${local.instance_alias}-sqs-queue" ],
+               [ "AWS/SQS", "ApproximateNumberOfMessagesNotVisible", "QueueName", "${local.instance_alias}-sqs-queue" ]
+              ],
+             "period":300,
+             "stat":"Average",
+             "region": "${var.aws_region}",
+             "title": "Shunting Yard SQS Queue Size Metrics"
+           }
+       },
+       {
+          "type":"metric",
+          "width":12,
+          "height":6,
+          "properties":{
+             "metrics": [
+               [ "AWS/SQS", "NumberOfMessagesDeleted", "QueueName", "${local.instance_alias}-sqs-queue" ]
+             ],
+             "period":300,
+             "stat":"Average",
+             "region": "${var.aws_region}",
+             "title": "Shunting Yard SQS Deleted Messages"
+          }
+       },
+       {
+          "type":"metric",
+          "width":12,
+          "height":6,
+          "properties":{
+             "metrics": [
+                 [ "AWS/SQS", "ApproximateAgeOfOldestMessage", "QueueName", "${local.instance_alias}-sqs-queue" ]
+             ],
+             "period":300,
+             "stat":"Average",
+             "view": "singleValue",
+             "region": "${var.aws_region}",
+             "title": "Shunting Yard SQS Age of Oldest Message (s)"
+           }
+       }  
+EOF
+}
+
+resource "aws_cloudwatch_dashboard" "beekeeper" {
+  dashboard_name = "${local.instance_alias}-${var.aws_region}"
+
+  dashboard_body = <<EOF
+ {
+   "widgets": [
+${join("", data.template_file.ecs_widgets.*.rendered)}
+${join("", data.template_file.sqs_widgets.*.rendered)}
+   ]
+ }
+ EOF
+}
+
+locals {
+  alerts = [
+    {
+      alarm_name  = "${local.instance_alias}-cpu"
+      namespace   = "AWS/ECS"
+      metric_name = "CPUUtilization"
+      threshold   = "90"
+    },
+    {
+      alarm_name  = "${local.instance_alias}-memory"
+      namespace   = "AWS/ECS"
+      metric_name = "MemoryUtilization"
+      threshold   = "80"
+    }
```

Comment:
Actually we need 4:

    {
      alarm_name  = "${local.instance_alias}-path-scheduler-cpu"
      namespace   = "AWS/ECS"
      metric_name = "CPUUtilization"
      threshold   = "90"
    },
    {
      alarm_name  = "${local.instance_alias}-path-scheduler-memory"
      namespace   = "AWS/ECS"
      metric_name = "MemoryUtilization"
      threshold   = "80"
    }
    {
      alarm_name  = "${local.instance_alias}-cleanup-cpu"
      namespace   = "AWS/ECS"
      metric_name = "CPUUtilization"
      threshold   = "90"
    },
    {
      alarm_name  = "${local.instance_alias}-cleanup-memory"
      namespace   = "AWS/ECS"
      metric_name = "MemoryUtilization"
      threshold   = "80"
    }